script/packet

```diff
@@ -20,4 +20,8 @@ cp -r ~/.ssh/. $path/cluster/ssh
 kubectl create configmap kubeconfig --from-file=$path/cluster/.kube/admin.conf -n litmus
 kubectl get pods --all-namespaces
 kubectl get nodes
+echo "--------------Attaching disks to the nodes------------------"
+cd ../packet-storage
+ansible-playbook create-volume.yml -vv --extra-vars "packet_api=$packet_api_token"
+cp -r /tmp $path/cluster/
```

Comment:
Don't copy the total content of /tmp it might contain more things, copy the directory only i.e "packet" to $path/cluster/
like: cp -r /tmp/packet $path/cluster/

Reply:
Copy script is not required!! just move the create disk line just after cluster crrate